CHANGES.rst

```diff
@@ -60,6 +60,10 @@ New Features
 
 - ``astropy.utils``
 
+  - New decorator: ``deprecated_renamed_argument``. This can be used when
+    renaming a function argument, but still want to allow for use of
+    the older parameter name. [#5214]
+
```

Comment:
Could I suggest: "This can be used to rename a function argument, while it still allows for the use of the older argument name."